packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -291,23 +297,36 @@ export default {
         placeOrder: async ({ commit, state }, {
             url, data, timeout
         }) => {
-            const authHeader = state.authToken && `Bearer ${state.authToken}`;
+            try {
+                const authHeader = state.authToken && `Bearer ${state.authToken}`;
 
-            const config = {
-                headers: {
-                    'Content-Type': 'application/json;v=2',
-                    'x-je-application-id': 7, // Responsive Web
-                    'x-je-application-version': applicationVerion,
-                    Authorization: authHeader
-                },
-                timeout
-            };
+                const config = {
+                    headers: {
+                        'Content-Type': 'application/json;v=2',
+                        'x-je-application-id': 7, // Responsive Web
+                        'x-je-application-version': applicationVerion,
+                        Authorization: authHeader
+                    },
+                    timeout
+                };
 
-            const response = await axios.post(url, data, config);
+                const response = await axios.post(url, data, config);
 
-            const { orderId } = response.data;
+                const { orderId } = response.data;
 
-            commit(UPDATE_ORDER_PLACED, orderId);
+                commit(UPDATE_ORDER_PLACED, orderId);
+                commit(UPDATE_ERRORS, []);
+            } catch (error) {
+                if (error.response && error.response.data) {
+                    const { errorCode } = error.response.data;
+
+                    const checkoutIssue = getIssueByCode(errorCode);
+
+                    commit(UPDATE_ERRORS, (checkoutIssue ? [checkoutIssue] : []));
+                }
+
+                throw error; // Handled by the calling function.
```

Comment:
No need for this comment, it's implicit that the error will be handled/rethrown upstream.

Reply:
I'm not a huge fan of comments, personally, but I think in this case it's a useful one. It's not implicit that an error will be handled upstream. If it's not handled, it can crash the application, or throw an unhandled exception to the user. With this comment, my intention was to inform other devs that we should be fine, since the calling function is indeed handling it.
Again, I'm more than happy to remove the comment if people think it's not needed, of course.